addons/rook/template/testgrid/k8s-docker.yaml

```diff
@@ -69,19 +77,23 @@
     minio_object_store_info
     validate_testfile rwtest minio.txt
     validate_read_write_object_store postupgrade minioupgrade.txt
+  unsupportedOSIDs:
+    - ubuntu-2204
 
 - name: Upgrade from 1.5.10
   installerSpec:
     kubernetes:
       version: "latest"
     weave:
       version: "latest"
-    docker:
+    containerd:
       version: "latest"
     minio:
       version: "latest"
     kotsadm:
       version: "latest"
+    ekco:
+      version: "latest"
```

Comment:
Do we still have any tests without ekco?

Reply:
No but I'd like to say at this point ekco is required and eventually make it core